### DISCUSSION TITLE
Penjelasan nilai minim pada AIC weights dan BIC weights pada compare_performance

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#122] [Q&A] [answered]
Pada suatu hasil compare_performance, saya melihat nilai AIC weights dan BIC weights yang kecil dan ada simbol < nya. 
Itu maksudnya apa ya? Kalau untuk weightsnya apakah lebih baik kecil atau besar?
![Capture](https://user-images.githubusercontent.com/95233522/171084713-796732b1-ddaa-4ee5-9784-4bf91793bc4c.JPG)

**Comment by muhrisman** [comment, marked as answer]
simbol <0.001 tersebut menandakan nilainya yang lebih kecil dari 0.001. Karena nilai yang diakomodir adalah 3 angka di belakang koma maka alternatifnya akan mengeluarkan notasi < untuk menandakan nilainya yang lebih kecil dari 0.001. 
Untuk nilainya sendiri akan lebih baik jika nilai BIC semakin kecil.

Semoga membantu :)

**Reply by Hertzspr** [reply]
Terima kasih mas, kalo AIC dan BICnya semakin kecil semakin baik, setuju, nah kalau weights nya?

**Reply by muhrisman** [reply]
Untuk weight ada baiknya memilih nilai yang lebih rendah ya pak :)

**Reply by Hertzspr** [reply]
Terima kasih.
